var/spack/repos/builtin/packages/mesa-demos/package.py

```diff
@@ -3,26 +3,61 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import sys
+
 from spack import *
 
 
 class MesaDemos(AutotoolsPackage):
     """This package provides some demo applications for testing Mesa."""
 
     homepage = "https://www.mesa3d.org"
-    url      = "https://github.com/freedesktop/mesa-demos/archive/mesa-demos-8.3.0.tar.gz"
+    url      = "https://github.com/freedesktop/mesa-demos/archive/mesa-demos-8.3.0.tar.gz"  # Should this be from free desktop gitlab?
```

Comment:
Development is essentially halted so it's last home of GH is probably correct so the comment can be dropped.